clients/client/src/main/java/org/projectnessie/client/http/HttpRequest.java

```diff
@@ -102,14 +112,15 @@ private HttpResponse executeRequest(Method method, Object body) throws HttpClien
       ResponseContext responseContext = new ResponseContextImpl(con);
       try {
         requestFilters.forEach(a -> a.filter(context));
+        putHeader("Accept", accept, headers);
         headers.entrySet().stream()
             .flatMap(e -> e.getValue().stream().map(x -> new SimpleImmutableEntry<>(e.getKey(), x)))
             .forEach(x -> con.setRequestProperty(x.getKey(), x.getValue()));
         con.setRequestMethod(method.name());
         if (method.equals(Method.PUT) || method.equals(Method.POST)) {
           // Need to set the Content-Type even if body==null, otherwise the server responds with
           // RESTEASY003065: Cannot consume content type
-          con.setRequestProperty("Content-Type", "application/json; charset=utf-8");
+          con.setRequestProperty("Content-Type", contentsType + "; charset=utf-8");
```

Comment:
i think `; charset=utf-8` should be part of the static above.

Reply:
updated